docs/API/V1/server_config.md

```diff
@@ -129,9 +128,33 @@ role.
 
 ## Response body
 
-Just as in [GET](#get-apiv1serverconfigurationkey), the `application/json`
-response body will be a JSON object showing the current (new) value of the
-modified server configuration settings.
+The `application/json` response body for `PUT` is exactly the same as for
+[`GET`](#response-body) when the same server configuration settings are
+requested, showing only the server configuration settings that were changed
+in the `PUT`.
+
+```
+PUT /api/v1/server/configuration/dataset-lifetime?value=4
+
+RESPONSE={
+    "dataset-lifetime": "4"
+}
+```
+
+or 
+
+```
+PUT /api/v1/server/configuration
+REQUEST={
+    "dataset-lifetime": "4 days",
+    "server-state": {"status": "enabled"}
+}
+
+RESPONSE={
+    "dataset-lifetime": "4",
+    "server-state": {"status": "enabled"}
+}
```

Comment:
Good question.

It was actually modeled on a pseudo-standard Python duration format, but it's not really supported anywhere, and it differs from the ISO 8601 duration format. I briefly considered changing to ISO 8601, but there's no conversion support for that in Python or Javascript (until Temporal).

In the wake of that I considered just supporting integer days instead of the more flexible input format, but decided not to bother removing the code.

The other option would be to write my own code to convert ISO 8601 durations, but the dashboard would need to do that as well. Leaving it as an integer seemed easier.

E.g., if I were to handle just days in ISO 8601 it would be "P<n>D"